src/ontology/uberon.Makefile

```diff
@@ -1025,147 +1095,144 @@ subsets/%-view.owl: uberon.owl contexts/context-%.owl tmp/simple-slim-seed.txt
-# and Owltools' source code, especially the following classes:
-# - <https://github.com/owlcollab/owltools/blob/master/OWLTools-Core/src/main/java/owltools/mooncat/SpeciesMergeUtil.java>
-# - <https://github.com/owlcollab/owltools/blob/master/OWLTools-Core/src/main/java/owltools/mooncat/EquivalenceSetMergeUtil.java>
-TAXON_GCI_RELS = RO:0002202 RO:0002496 RO:0002497 BFO:0000051
-$(TMPDIR)/unreasoned-composite-%.owl: $(TMPDIR)/stripped-composite-%.owl
-	$(OWLTOOLS) $< \
-		    --reasoner elk \
-		    --merge-species-ontology -s 'mouse'      -t NCBITaxon:10090 -q $(TAXON_GCI_RELS) \
-		    --merge-species-ontology -s 'human'      -t NCBITaxon:9606  -q $(TAXON_GCI_RELS) \
-		    --merge-species-ontology -s 'primate'    -t NCBITaxon:9443 \
-		    --merge-species-ontology -s 'Xenopus'    -t NCBITaxon:8353 \
-		    --merge-species-ontology -s 'Danio'      -t NCBITaxon:7954 \
-		    --merge-species-ontology -s 'Drosophila' -t NCBITaxon:7227 \
-		    --merge-species-ontology -s 'C elegans'  -t NCBITaxon:6237 \
-		    --merge-equivalence-sets -s UBERON 10 -s CL 9 -s CARO 5 \
-		                             -l UBERON 10 -l CL 9 \
-		                             -d UBERON 10 -d CL 9 \
-		    -o -f ofn $@
-.PRECIOUS: $(TMPDIR)/unreasoned-composite-%.owl
-
-# Step 3: Standard reasoning step
-composite-%.owl: $(TMPDIR)/unreasoned-composite-%.owl
-	$(ROBOT) reason -i $< -r ELK --equivalent-classes-allowed all \
-		 relax \
-		 reduce -r ELK \
-		 annotate --ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) \
-		          -o $@.tmp.owl && mv $@.tmp.owl $@
-.PRECIOUS: composite-%.owl
-
-# ODK insanity: This rule is EXACTLY the same as the rule above,
-# composite-metazoan does not require any special treatment compared to
-# the other composite-% products. But because the ODK-generated Makefile
-# defines a non-implicit rule with that target, we MUST override it with
-# a non-implicit rule -- the implicit rule above is not enough. So we
-# have no other choice but to duplicate the rule. Note to future
-# maintainers: any change to "step 3" rule above MUST be replicated in
-# this rule as well!
-composite-metazoan.owl: $(TMPDIR)/unreasoned-composite-metazoan.owl
-	$(ROBOT) reason -i $< -r ELK --equivalent-classes-allowed all \
-		 relax \
-		 reduce -r ELK \
-		 annotate --ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) \
-		          -o $@.tmp.owl && mv $@.tmp.owl $@
 
-# Likewise.
-composite-vertebrate.owl: $(TMPDIR)/unreasoned-composite-vertebrate.owl
-	$(ROBOT) reason -i $< -r ELK --equivalent-classes-allowed all \
+# Step 1: Create a "collected" ontology, which is simply a merge of
+# Uberon, CL, the CARO bridges, and the components listed in the
+# COLLECTED_*_SOURCES variables above.
+.PRECIOUS: $(TMPDIR)/collected-%.owl
+$(TMPDIR)/collected-%.owl: $(BRIDGEDIR)/collected-%-hdr.owl uberon.owl $(IMPORTDIR)/local-cl.owl \
+		 $(BRIDGEDIR)/uberon-bridge-to-caro.owl \
+		 $(BRIDGEDIR)/cl-bridge-to-caro.owl \
```

Comment:
Given that there has been an effort to get rid of caro in favour of a thin anatomy top layer in uberon, https://github.com/obophenotype/uberon/pull/2614, can you judge if removing this will cause issues?

Reply:
Ok, just wanted you to be aware of this, since you are, I will resolve comment!